src/transformers/__init__.py

```diff
@@ -22,7 +22,7 @@
 from .configuration_auto import ALL_PRETRAINED_CONFIG_ARCHIVE_MAP, CONFIG_MAPPING, AutoConfig
 from .configuration_bart import BartConfig
 from .configuration_bert import BERT_PRETRAINED_CONFIG_ARCHIVE_MAP, BertConfig
-from .configuration_causal_bert import CausalBertConfig
```

Comment:
This class has been in a release already. We can't remove it without proper deprecation warnings.

Reply:
This was a weird diff -> the config, tokenizer and model `CausalBert...` were never in the library - I added them yesterday to the PR. If you look at the changed files now you can see that no previous model names are removed :-)

Reply:
Will still need 1,2 days to fiinish the PR including integration tests and model cards, etc...so no need to review yet :-)

Reply:
Oh then in that case, no problem with renaming things :-)